components/input.tsx

```diff
@@ -2,36 +2,65 @@ import clsx from "clsx";
 import React from "react";
 
 type InputTextProps = {
-  label?: string;
+  label: string;
   error?: string;
+  id: string;
+  suffix?: string;
+  size?: "sm" | "md" | "lg";
   //TODO define if we want variants with icons/prefixes/suffixes
 } & React.InputHTMLAttributes<HTMLInputElement>;
 
 const InputText = React.forwardRef<HTMLInputElement, InputTextProps>(
-  ({ label, required, error, id, ...props }, ref) => {
+  ({ label, required, error, id, suffix, size = "md", ...props }, ref) => {
+    const labelAndSuffixStyle =
+      "flex items-center justify-center bg-primary-950 px-2 text-xs text-primary-100";
+
+    const textSize = (() => {
```

Comment:
We should change the size of the input, in order to make it look bigger or smaller based on size.   (ex. using different paddings for each size, like in the button ).

Composing the classes through a function doesn't  take full advantage of the intellisense

Reply:
yeah but we have advantages on code cleaning :)